app/admin/classes/AdminController.php

```diff
@@ -98,7 +98,7 @@ public function __construct()
             $manager->bindToController();
         }
 
-        $this->fireEvent('controller.afterConstructor', [$this]);
+        event(new \Admin\Events\Controller\AfterConstructor($this));
```

Comment:
Instead of using fully qualified class names, import the classes instead...

Reply:
Its easier to read :)

Reply:
All updated -  let me know how you want to handle return values on events so I can deal with the fireSystemEvent calls